docs/src/getting-started/installation.md

```diff
@@ -329,6 +330,76 @@ Make sure to include this section in `Scarb.toml`
 test = "snforge test"
 ```
 
+## Set up shell completions (optional)
+
+<details>
+  <summary><strong>Bash</strong></summary>
+
+Completions are configured by doing the following:
+```shell
+mkdir -p "${ASDF_DATA_DIR:-$HOME/.asdf}/completions"
+sncast completion bash > "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/sncast.bash"
+snforge completion bash > "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/snforge.bash"
+```
+
+Then add the following to your `.bash`:
+```shell
+# source completion scripts
+. "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/sncast.bash"     
+. "${ASDF_DATA_DIR:-$HOME/.asdf}/completions/snforge.bash"
+```
```

Comment:
Should it be a bullet-point list?

<img width="555" alt="image" src="https://github.com/user-attachments/assets/c30f7d68-a255-4c4a-a5b6-6d9bb6b44961" />